pepper-apis/src/main/java/org/broadinstitute/ddp/util/ElasticsearchServiceUtil.java

```diff
@@ -46,12 +51,18 @@ public static RestHighLevelClient getClientForElasticsearchCloud(Config cfg) thr
                 new UsernamePasswordCredentials(userName, password));
 
         URL url = new URL(cfg.getString(ConfigFile.ELASTICSEARCH_URL));
-        LOG.info("Creating Elasticsearch client with URL: {}", url);
+        LOG.info("Using Elasticsearch client URL: {}", url);
 
         String proxy = ConfigUtil.getStrIfPresent(cfg, ConfigFile.ELASTICSEARCH_PROXY);
         final URL proxyUrl = StringUtils.isNotBlank(proxy) ? new URL(proxy) : null;
         if (proxyUrl != null) {
-            LOG.info("Using Elasticsearch client proxy: {}", proxy);
+            LOG.info("Using Elasticsearch client proxy URL: {}", proxy);
+        }
+
+        int key = Objects.hash(url, proxyUrl, userName);
```

Comment:
We have multiple connections with different urls, user names, etc or is this thinking ahead?

Reply:
We don't, but given how the code here is written I think this arrangement fits with the surrounding code better.